src/lib/input/input-container.ts

```diff
@@ -457,6 +457,7 @@ export class MdInputContainer implements AfterViewInit, AfterContentInit, AfterC
 
   /** Reference to the input's underline element. */
   @ViewChild('underline') underlineRef: ElementRef;
+  @ViewChild('connectionContainer') _connectionContainerRef: ElementRef;
```

Comment:
Can you not use the `underlineRef`? The datepicker is using it for a similar purpose.